### PR TITLE
Add crash revision when regression range is not available.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
unsure if we should add crash revision everytime. if you feel that, i can add that always.

File: src/appengine/handlers/revisions_info.py [hunk 1]
```diff
@@ -25,13 +25,25 @@ class Handler(base_handler.Handler):
...
+    revision = self.request.get('revision')
+    if revision:
+      if not revision.isdigit():
+        raise helpers.EarlyExitException('Revision is not an integer.', 400)
+      start_revision = end_revision = revision
+    else:
+      revision_range = self.request.get('range')
+      if revision_range:
+        try:
+          [start_revision, end_revision] = revision_range.split(':')
```
Comment (oliverchang) [inline src/appengine/handlers/revisions_info.py]: nit: [] should be unnecessary here.

**Pull request merged**